.storybook/components/DesignTokens/Tier2/Forms.jsx

```diff
@@ -8,42 +8,36 @@ export class Tier2Forms extends Component {
     return (
       <Section title="Form Border">
         <Grid>
-          {filterTokens(`eds-theme-form-border-width`).map(function (listItem) {
-            return (
-              <GridItem key={listItem.name}>
-                <TokenSpecimen
-                  inlineStyles={{
-                    backgroundColor: 'transparent',
-                    borderWidth: `var(${listItem.name})`,
-                    borderStyle: 'solid',
-                    borderColor: 'black',
-                  }}
-                  name={listItem.name}
-                  value={listItem.value}
-                />
-              </GridItem>
-            );
-          })}
+          {filterTokens(`eds-theme-form-border-width`).map((listItem) => (
```

Comment:
Cleaned up this one b/c the formatting was getting garbled by prettier in some weird way. All of these could be converted into MDX files instead of having the class component wrapper and storybook file in the end